### PR TITLE
WIP: add delete buttons to GUI

### PR DESCRIPTION
**Pull request by antgonza**
All JS and HMTL is done. Next add calls/connections to python

### CODE REVIEW

File: qiita_pet/handlers/api_proxy/prep_template.py [hunk 1]
```diff
@@ -158,8 +158,8 @@ def prep_template_ajax_get_req(user_id, prep_id):
             download_qiime.append(fp_id)
         else:
             download_prep.append(fp_id)
-    download_prep = download_prep[0]
-    download_qiime = download_qiime[0]
+    download_prep = download_prep[0] if download_prep else None
```
Comment (josenavas) [inline qiita_pet/handlers/api_proxy/prep_template.py]: @antgonza why is this change needed? In which situation are these lists empty?

Comment (antgonza) [inline qiita_pet/handlers/api_proxy/prep_template.py]: To be 100% honest I can't remember exactly when it happens but it happens. I think it's when the QIIME map file can't be created.  If you check the logs you are going to see a few of these:
```python
    HTTPRequest(protocol='https', host='qiita.ucsd.edu', method='GET', uri='/study/description/prep_template/?prep_id=1748&study_id=10512', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Accept-Language': 'en-US,en;q=0.8,es;q=0.6', 'Accept-Encoding': 'gzip, deflate, sdch, br', 'X-Forwarded-For': '73.217.94.32', 'Connection': 'upgrade', 'Accept': '*/*', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36', 'Host': 'qiita.ucsd.edu', 'X-Requested-With': 'XMLHttpRequest', 'Referer': 'https://qiita.ucsd.edu/study/description/10512', 'X-Real-Ip': '73.217.94.32', 'Cookie': 'jlinkauthserver=davis; jlinksessionidx=z5109caad5d96a7e441999e7a96f20c28; jlinkappx=/employeeMyTimeEntry/calendar-time-entry.htm; xjlinkloginMyTimeEntry.User=OK; __utma=57960238.431222674.1463057811.1463410241.1463675783.2; __utmc=57960238; __utmz=57960238.1463410241.1.1.utmcsr=a4.ucsd.edu|utmccn=(referral)|utmcmd=referral|utmcct=/tritON/profile/SAML2/Redirect/SSO; user="ImFudG9uaW9nQHVjc2QuZWR1Ig==|1463762818|947f73f203ee2fbc2df70561c09076fe0c5e7588"; _ga=GA1.2.431222674.1463057811'})
    Traceback (most recent call last):
      File "/home/qiita/.virtualenvs/qiita/lib/python2.7/site-packages/tornado/web.py", line 1141, in _when_complete
        callback()
      File "/home/qiita/.virtualenvs/qiita/lib/python2.7/site-packages/tornado/web.py", line 1162, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/qiita/.virtualenvs/qiita/lib/python2.7/site-packages/tornado/web.py", line 2297, in wrapper
        return method(self, *args, **kwargs)
      File "/home/qiita/qiita_main/qiita_pet/handlers/study_handlers/prep_template.py", line 58, in get
        res = prep_template_ajax_get_req(self.current_user.id, prep_id)
      File "/home/qiita/qiita_main/qiita_pet/handlers/api_proxy/prep_template.py", line 162, in prep_template_ajax_get_req
        download_qiime = download_qiime[0]
    IndexError: list index out of range
```

Comment (josenavas) [inline qiita_pet/handlers/api_proxy/prep_template.py]: That is more concerning as that scenario should never happen. I don't think the solution is add the if statement here, but rather debug and find in which situation the QIIME map is not generated.

Comment (antgonza) [inline qiita_pet/handlers/api_proxy/prep_template.py]: Wanna open an issue?

Comment (josenavas) [inline qiita_pet/handlers/api_proxy/prep_template.py]: Done: https://github.com/biocore/qiita/issues/1916

**Comment by josenavas** [comment]
Closing in favor of: https://github.com/biocore/qiita/pull/1917
